client/src/components/layout/Question.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import { Typography } from '@material-ui/core';
+
+import ReactMarkdown from 'react-markdown';
+import {Prism as SyntaxHighlighter} from 'react-syntax-highlighter';
+import gfm from 'remark-gfm';
```

Comment:
This import is failing because it's not in the package.json

Reply:
The missing packages have been already added to the package.json file